src/components/common/Footer/FooterLinks.tsx

```diff
@@ -98,7 +97,7 @@ function LinksList({ items }: LinksListProps) {
     <UIList>
       {items.map((item) => (
         <li key={item.name}>
-          <Link variant="footer" href={item.href}>
+          <Link variant="footer" to={item.href}>
```

Comment:
Actually, we are not sure if those footer links will be internal or external links right?
Hmmm, how about we add this observation on our [readme – Performance & QA section](https://github.com/vtex-sites/base.store#-performance--qa)